darshan-util/pydarshan/docs/usage.rst

```diff
@@ -1,41 +1,79 @@
+.. _usage:
+
 =====
 Usage
 =====
 
+Darshan job summary tool
+------------------------
 
-Darshan Report Object Interface
--------------------------------
+As a starting point, users can use PyDarshan to generate detailed
+summary HTML reports of I/O activity for a given Darshan log.
+Usage of this job summary tool is described below. ::
 
-To get started with PyDarshan you can simply use::
+    usage: darshan <command> summary [-h] [--output OUTPUT] [--enable_dxt_heatmap] log_path
```

Comment:
What is `<command>` doing here? I genuinely don't know what you'd fill in there, maybe delete even if the help prints that for whatever reason?